optaplanner-examples/src/test/java/org/optaplanner/examples/app/AllExamplesSolutionImporterTest.java

```diff
@@ -0,0 +1,55 @@
+package org.optaplanner.examples.app;
+
+import static java.util.stream.Collectors.*;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.apache.commons.codec.digest.DigestUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class AllExamplesSolutionImporterTest {
```

Comment:
How much time does this test take?

Reply:
Not too long, surprisingly. A couple of seconds. Barely noticeable on my machine.